.circleci/config.yml

```diff
@@ -88,6 +88,64 @@ jobs:
       - store_artifacts:
           path: .circleci/artifacts.html
           destination: artifacts.html
+
+  build-python36:
+    machine:
+      image: ubuntu-1604:201903-01
+    working_directory: ~/sync-engine
+
+    steps:
+      - checkout
+
+      - run:
+          name: build sync-engine image
+          command: docker-compose build --build-arg PYTHON_VERSION=3.6 app || true
```

Comment:
Why do we need the `|| true` here? Shouldn't the process halt if building the image fails?

Reply:
Oh I get it. No worries, makes total sense.